html/images/os/allied.svg

```diff
@@ -0,0 +1 @@
+alliedtelesis.svg
```

Comment:
Looks like the svg has gone wrong.

Reply:
Nah - this is a soft symlink to alliedtelesis.svg. 

I saw no reason to duplicate the svg file for multiple OSes, so used a symlink to do this.

Reply:
Then just have on image and change the icon the yaml discovery file to point to that one image :)

Reply:
Ah yeah fair comment.